notify_user.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+import sys
+import json
+import rc_util
+import smtplib
+import dataset
+from rc_rmq import RCRMQ
+from jinja2 import Template
+from datetime import datetime
+import mail_config as mail_cfg
+
+task = 'notify_user'
+
+args = rc_util.get_args()
+logger = rc_util.get_logger(args)
+
+db = dataset.connect(f'sqlite:///.agent_db/{task}.db')
+table = db['notified_user']
+
+# Instantiate rabbitmq object
+rc_rmq = RCRMQ({'exchange': 'RegUsr', 'exchange_type': 'topic'})
+
+# Email instruction to user
+def notify_user(ch, method, properties, body):
+    msg = json.loads(body)
+    username = msg['username']
+    user_email = msg['email']
+    msg['task'] = task
+    msg['success'] = False
+
+    try:
+
+        # Search username in database
+        record = table.find_one(username=username)
+
+        if record:
+            # Update counter
+            count = record['count']
+            table.update({'username': username, 'count': count + 1}, ['username'])
+
+            logger.debug(f'User {username} counter updated to {count + 1}')
+
+        else:
+            # Send email to user
+            receiver = [user_mail, mail_cfg.My_mail]
+            message = Template(mail_cfg.Whole_mail).render(username=username, to=user_email)
+
+            if args.dry_run:
+                logger.info(f'smtp = smtplib.SMTP({mail_cfg.Server})')
+                logger.info(f'smtp.sendmail({sender}, {receiver}, message)')
+                logger.info(f"table.insert({'username': {username}, 'count': 1, 'sent_at': {datetime.now()}})")
+
+            else:
+                smtp = smtplib.SMTP(mail_cfg.Server)
+                smtp.sendmail(sender, receiver, message)
```

Comment:
Similar to the change on line 50, this should be - `smtp.sendmail(mail_cfg.Sender, receiver, message)`

Reply:
should be fixed in 5ecfce5